src/main/resources/io/github/dfdlschemas/envelopepayload/xsd/envelopePayload.dfdl.xsd

```diff
@@ -0,0 +1,94 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<schema
+  xmlns="http://www.w3.org/2001/XMLSchema"
+  xmlns:xs="http://www.w3.org/2001/XMLSchema"
+  xmlns:fn="http://www.w3.org/2005/xpath-functions"
+  xmlns:dfdl="http://www.ogf.org/dfdl/dfdl-1.0/"
+  xmlns:ep="urn:io.github.dfdlschemas.envelopepayload"
+  targetNamespace="urn:io.github.dfdlschemas.envelopepayload"
+  xmlns:msHdr="urn:milstd2045DFDL"
+  xmlns:pcap="urn:pcap:2.4"
+  elementFormDefault="unqualified">
+
+  <import namespace="urn:milstd2045DFDL"
+          schemaLocation="/com/owlcyberdefense/mil-std-2045/xsd/milstd2045_application_header.dfdl.xsd"/>
+  <import namespace="urn:pcap:2.4"
+          schemaLocation="/com/tresys/pcap/xsd/pcap.dfdl.xsd"/>
+
+  <annotation>
+    <appinfo source="http://www.ogf.org/dfdl/">
+      <dfdl:format ref="msHdr:msCommon"/>
+      <dfdl:defineVariable name="filename" type="xs:string"/>
+      <dfdl:defineVariable name="binaryFileKind" type="xs:string"/>
+    </appinfo>
+  </annotation>
+
+  <group name="payloadContentGroup">
```

Comment:
`<!-- We will use this payloadContentGroup group to define a tcpContentType complex type in another schema file -->`